### DISCUSSION TITLE
Fetch/Extract Full Content By Default

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#639] [Feature Requests]
Sticky Toggle works well and as intented but I feel like in some cases it could be useful to load all articles in full length, perhaps by feed and/or tag.

This could be particularly useful when traveling to a place expecting to have bad or no internet connection(like a holiday trip).

Im not sure if this operation would be too expensive or inconvenient but I thought I would express it nonetheless.

Anyway, thanks for making this app and happy Holidays.

Not suee if duplicate of #550

**Comment by jocmp** [comment]
Hey @299792458ms - this does seem related to #550 (I'm also tracking this via https://github.com/jocmp/capyreader/issues/560).The biggest hurdles will be what you mentioned: expensive network calls and clarity on how the feature works. I think it's possible, and I definitely want to add it at some point in the future.

I think configuration would work by feed, maybe follow the sticky content, and full articles would stay saved until the entire article is deleted.

**Reply by ghost** [reply]
Thanks for considering!

**Comment by memdealer** [comment]
Yes, please! 

I often use RSS feeds while traveling (on trains or planes) with little to no internet access. I can only see the article title without the body (e.g., Hacker News feeds), so I cannot fully use feeds that don’t include full text.

This feature would make offline reading - a thing back again.

PS:
Thanks for the app—its design and usability are fantastic! This has been a game-changer after struggling with apps like FeedMe and Reader5 emulators on Android. @jocmp 
 
I'm looking forward to seeing this feature added!
